fw/http_parser.c

```diff
@@ -2492,6 +2492,133 @@ __req_parse_transfer_encoding(TfwHttpMsg *hm, unsigned char *data, size_t len)
 	return __parse_transfer_encoding(hm, data, len, true, false);
 }
 
+static int
+__resp_parse_trailer(TfwHttpMsg *hm, unsigned char *data, size_t len)
```

Comment:
Seems we allow duplicated `Trailer` header, though RFC doesn't contain direct prohibitions this looks like confusion and we must not allow duplicated `Trailer` header.